packages/@core.next/auth.jsx

```diff
@@ -127,7 +133,7 @@ const AuthProvider = ({ children, initialUserValue }) => {
         <AuthContext.Provider
             value={{
                 isAuthenticated: !!user,
-                isLoading: userLoading || signinLoading || signoutLoading,
+                isLoading: isUserLoading || signinLoading || signoutLoading,
```

Comment:
maybe we need to remove `signinLoading || signoutLoading` ?

Reply:
These two variables are showing other loading states within this hook. I'd like to keep them here.